CDP4Reporting/Utilities/ReportZipArchive.cs

```diff
@@ -0,0 +1,56 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="ReportZipArchive.cs" company="RHEA System S.A.">
+//    Copyright (c) 2015-2020 RHEA System S.A.
+//
+//    Author: Sam Gerené, Alex Vorobiev, Alexander van Delft, Cozmin Velciu, Adrian Chivu
+//
+//    This file is part of CDP4-IME Community Edition.
+//    The CDP4-IME Community Edition is the RHEA Concurrent Design Desktop Application and Excel Integration
+//    compliant with ECSS-E-TM-10-25 Annex A and Annex C.
+//
+//    The CDP4-IME Community Edition is free software; you can redistribute it and/or
+//    modify it under the terms of the GNU Affero General Public
+//    License as published by the Free Software Foundation; either
+//    version 3 of the License, or any later version.
+//
+//    The CDP4-IME Community Edition is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//    GNU Affero General Public License for more details.
+//
+//    You should have received a copy of the GNU Affero General Public License
+//    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+
+namespace CDP4Reporting.Utilities
+{
+    using System;
+    using System.IO;
+
+    /// <summary>
+    /// Struct that maps streams on archive zip file path
+    /// </summary>
+    public struct ReportZipArchive : IDisposable
+    {
+        /// <summary>
+        /// The report file <see cref="Stream"/>
+        /// </summary>
+        public Stream Repx { get; set; }
```

Comment:
what is the meaning of the `Repx`? can we please use a name that conveys meaning?

Reply:
Repx is the extension of a DevExpress report file. We created Rep4 as an extension which is a zip archive that contains a Report definition and a Datasource file (*.repx and *.ds). That's why repx was used. I changed to ReportDefinition.